sdk/src/main/java/com/talkable/sdk/models/Origin.java

```diff
@@ -19,6 +21,15 @@
     String[] campaignTags = {};
     String trafficSource;
     @SerializedName("ip_address") String ipAddress = "current";
+    @SerializedName("custom_properties") HashMap<String, String> customProperties;
```

Comment:
I think it should be moved to `Customer` model as in case we need a deserialization it should be in `Customer` model.